src/battle_script_commands.c

```diff
@@ -2111,7 +2111,9 @@ static void Cmd_adjustdamage(void)
     }
     if (gSpecialStatuses[gBattlerAttacker].gemBoost
         && !(gMoveResultFlags & MOVE_RESULT_NO_EFFECT)
-        && gBattleMons[gBattlerAttacker].item)
+        && gBattleMons[gBattlerAttacker].item
+        && gBattleMoves[gCurrentMove].effect != EFFECT_PLEDGE
```

Comment:
Only if a combined move is used.

Reply:
Not sure but both on showdown and gen5 gems didn't activate if pledge moves were used in isolation.

Reply:
Here's the proof:
![melonDS_s0Z7pglWl4](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/818fc994-7925-43f9-808e-df39647e3e46)

Reply:
Seems I read that wrong on Bulbapedia. But I now noticed that this wasn't implemented:
> If a combined move is used, it will always receive the [same-type attack bonus](https://bulbapedia.bulbagarden.net/wiki/Same-type_attack_bonus) no matter which Pokémon is the one executing it, or what type the combined move is.